core/src/Lender.sol

```diff
@@ -477,6 +487,6 @@ contract Lender is Ledger {
 
         totalSupply = cache.totalSupply.safeCastTo112();
         lastBalance = cache.lastBalance.safeCastTo112();
-        lastAccrualTime = uint32(block.timestamp); // Disables reentrancy guard
+        lastAccrualTime = uint32(block.timestamp); // Disables reentrancy guard if there was one
```

Comment:
Why keep this then?

Reply:
Even if we don't care about the reentrancy guard at all, we still want to update `lastAccrualTime` to the current block's timestamp, because we did, in fact, accrue interest (as part of `_load`)